bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/types/BPackage.java

```diff
@@ -35,18 +35,20 @@
     public String org;
     public String name;
     public String version;
-    private int hashCode = Objects.hash(org, name, version);
+    private int hashCode;
 
     public BPackage(String org, String name, String version) {
         this.org = org;
         this.name = name;
         this.version = version;
+        hashCode = Objects.hash(org, name, version);
     }
 
     public BPackage(String org, String name) {
         this.org = org;
         this.name = name;
         this.version = "";
+        hashCode = Objects.hash(org, name);
```

Comment:
With Type ID, this should contain a version.

Reply:
I didn't quite get you, for this particular constructor version is a constant right?

Reply:
One assumption in typeID design is, modules always have a version. Maybe we should remove this constructor.

Reply:
Ack. Will look into removing this in a subsequent PR.